index.js

```diff
@@ -100,23 +139,31 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
+
+function triple(el) {
```

Comment:
Try to use descriptive variables if you can!
While `el` technically does work, it's in your own interest to use variables with meaning like "number" or "name", etc.